### PR TITLE
cli: reindex only specified types, not every workflow

### PR DESCRIPTION
**Pull request by szymonlopaciuk**

### CODE REVIEW

File: invenio_workflows_ui/cli.py [hunk 1]
```diff
@@ -80,7 +80,8 @@ def reindex(yes_i_know, data_type, batch_size, queue_name):
 
     click.secho('Sending workflows to the indexing queue...', fg='green')
 
-    query = db.session.query(WorkflowObjectModel.id)
+    where_type = WorkflowObjectModel.data_type.in_(data_type)
+    query = db.session.query(WorkflowObjectModel.id).filter(where_type)
```
Comment (michamos) [inline invenio_workflows_ui/cli.py]: minor remark: I assume you're doing this in order not to have lines that are too long. One way around that is to do something like https://github.com/ambv/black#call-chains,
```
query = (db.session.query(WorkflowObjectModel.id)
         .filter(WorkflowObjectModel.data_type.in_(data_type))
```

Comment (szymonlopaciuk) [reply invenio_workflows_ui/cli.py]: Huh, didn't think of that. Neat!

Comment (david-caro) [reply invenio_workflows_ui/cli.py]: Personally I don't like that type of coding, as it also complicates a lot debugging (you loose all the intermediate objects that are generated by the intermediate calls). But again, personal preference.

Comment (michamos) [reply invenio_workflows_ui/cli.py]: In general I agree, in this specific case I don't, as this is one DB query, the intermediate `where_type` is not a call that one could debug, but a predicate in the SQLAlchemy DSL that gets translated to a piece of a SQL statement when executing the query. So there's little value in having it separately IMO.

**Pull request merged**